snippets/ava.sublime-snippet

```diff
@@ -1,9 +1,9 @@
 <snippet>
     <content><![CDATA[
 import test from 'ava';
-import $1 from '$2';
+import $1 from ${2:'${1/([A-Z])/-\l$1/g}'};
 
-test('${3:title}', t => {
+test(${3/(.+)|.*/?1:'/}${3:title}${3/(.+)|.*/?1:', /}t => {
```

Comment:
Hell no! Tiredness make me write unecessary complicated patterns. As you will see in the next commit, the final regex is wayyyy simpler. Thank you.